pom.xml

```diff
@@ -148,8 +148,8 @@ limitations under the License.
         </plugin>
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-surefire-plugin</artifactId>
-          <version>2.22.2</version>
+          <artifactId>maven-dependency-plugin</artifactId>
+          <version>3.1.2</version>
```

Comment:
Can this be split into a separate pr? In case we have to rollback timeout changes

Reply:
I will split this into a separate commit. I assume that should be fine? e.g. if we merge the PR and don't squash, then both commits will be there, and we can just revert the other commit if needed